implementation/src/main/java/io/smallrye/openapi/runtime/util/SchemaFactory.java

```diff
@@ -68,59 +111,157 @@ public static Schema readSchema(IndexView index,
         schema.setExternalDocs(readExternalDocs(annotation.value(OpenApiConstants.PROP_EXTERNAL_DOCS)));
         schema.setDeprecated((Boolean) overrides.getOrDefault(OpenApiConstants.PROP_DEPRECATED, JandexUtil.booleanValue(annotation, OpenApiConstants.PROP_DEPRECATED)));
         schema.setType((Schema.SchemaType) overrides.getOrDefault(OpenApiConstants.PROP_TYPE, JandexUtil.enumValue(annotation, OpenApiConstants.PROP_TYPE, Schema.SchemaType.class)));
-        schema.setEnumeration((List<Object>) overrides.getOrDefault(OpenApiConstants.PROP_ENUM, JandexUtil.stringListValue(annotation, OpenApiConstants.PROP_ENUM)));
+        schema.setEnumeration((List<Object>) overrides.getOrDefault(OpenApiConstants.PROP_ENUMERATION, JandexUtil.stringListValue(annotation, OpenApiConstants.PROP_ENUMERATION)));
```

Comment:
Good catch - the OpenAPI data model uses `enum` for this, which is probably why this was wrong.  I guess there is/was no test coverage for this. :(

Reply:
This actually caused the TCK to fail with the consolidated version of this method. I think you're right that none of the tests within the implementation project hit this.